subsys/nrf_security/Kconfig

```
@@ -32,13 +32,13 @@ config NORDIC_SECURITY_BACKEND
 
 config NRF_SECURITY
 	bool
-	prompt "Use nrf_security module" if !PSA_PROMPTLESS
+	prompt "Use nrf_security" if !PSA_PROMPTLESS
```

I'm wondering if we should change the wording to "Enable" here.

```suggestion
	prompt "Enable nRF Security" if !PSA_PROMPTLESS
```

---

Sure, it's more in line with other config options as well.